docs/conf.py

```diff
@@ -300,7 +300,7 @@ def source_replace(app, docname, source):
 
 # Dict of replacements.
 source_replacements = {
-    "{PLONE_BACKEND_VERSION}": "6.0.0b3",
+    "{PLONE_BACKEND_VERSION}": "6.0.0",
```

Comment:
From now on, we could always recommend `6.0` for the Backend version

Reply:
@stevepiercy: @ericof 's comment is about that he managed to set up Plone image in a way that the use can be done without mentioning the minor version.
As you merged this already, I'll open a new PR.